### PR TITLE
Added InitInstance for backup:PA-577

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
Added seperate InitInstance for Backup
**Which issue(s) this PR fixes** (optional)
Closes #
PA-577
**Special notes for your reviewer**:

Running all the testcases with postgres-backup,cassandra : https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test-byoc/29/consoleFull

### CODE REVIEW

File: tests/backup/backup_basic_test.go [hunk 1]
```diff
@@ -59,22 +61,55 @@ func TestBasic(t *testing.T) {
...
+		log.FailOnError(err, "Error occurred while Backup Driver Initialization")
+	}
+	// Getting Px version info
+	pxVersion, err := Inst().V.GetDriverVersion()
+	log.FailOnError(err, "Error occurred while getting PX version")
+	commitID := strings.Split(pxVersion, "-")[1]
+	t := Inst().Dash.TestSet
+	t.CommitID = commitID
+	if pxVersion != "" {
+		t.Tags["px-version"] = pxVersion
```
Comment (kshithijiyer-px) [inline tests/backup/backup_basic_test.go:94]: Any specific reason for maintaining pxVersion?

Comment (mkoppal-px) [reply tests/backup/backup_basic_test.go:94]: We need this so that we can keep a track of the px-version we are running on

**Review by mkoppal-px — APPROVED** [review]
LGTM if the tests are passing without any issues

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**